### DISCUSSION TITLE
Can not show logs of DAGS on UI when deploy on Kubernetes

### DISCUSSION THREAD

**Discussion by htthien22** [apache/airflow#30576] [Q&A] [answered]
I have deploy  airflow 2.3.3, 2.5.2 on Kubernetes 1.19.5 .
I can view logs in /opt/logs... but logs of DAGS not show on UI.
I don't know what happened on K8S cluster, I tried on docker-compose and it shows normal logs.

![2](https://user-images.githubusercontent.com/49353792/231117455-5399897f-7941-4d33-862e-f5306ebcc60b.png)

![1](https://user-images.githubusercontent.com/49353792/231117494-08c114c5-04e9-4510-87d5-1d2042e8071a.png)

**Can anyone help?**

Thanks in advance

**Comment by kobethuwis** [comment]
Can you share your chart configuration?

**Reply by htthien22** [reply, marked as answer]
Thanks for the reply, I found the cause: Before on a workload I run two pods(webserver and scheduler) After I split into two workloads(webserver <> scheduler) and problem solved.
